Sources/ApodiniAuthorization/Delegates/AuthorizationRequirementsChecker.swift

```diff
@@ -38,7 +39,7 @@ struct AuthorizationRequirementsChecker<H: Handler, Element: Authenticatable>: H
         if !authenticatable.isAuthorized {
             switch type {
             case .optional:
-                return await try delegate.instance().handle()
+                return try await delegate.instance().handle()
```

Comment:
Nice that this is fixed now, I think i was getting some compiler warnings because of that :D

Reply:
Seems like I somehow messed that up back then 😅